.eslintignore

```diff
@@ -0,0 +1 @@
+!/.aws
```

Comment:
This directory contains typescript and should conform to the linting rules as well. I don't think we should ignore it

Reply:
ignore this, I did not notice the `!` sign